tests/data/generator/t_protocol/t_protocol.proto

```diff
@@ -98,12 +98,12 @@ message TProtocolMessage{
     string dialogue_responder_reference = 3;
     int32 target = 4;
     oneof performative{
-        Performative_Ct performative_ct = 5;
-        Performative_Empty_Contents performative_empty_contents = 6;
-        Performative_Mt performative_mt = 7;
-        Performative_O performative_o = 8;
-        Performative_Pct performative_pct = 9;
-        Performative_Pmt performative_pmt = 10;
-        Performative_Pt performative_pt = 11;
+        Performative_Ct_performative performative_ct = 5;
```

Comment:
Seems like we should have `_Performative` to have consistent capitalization?

Reply:
Done